lib/App/index.js

```diff
@@ -311,6 +308,29 @@ class App {
 		this.debug(`Validated successfully`);
 		
 	}
+
+    _checkZwaveForSetting(driver, setting) {
+		if (!setting.zwave) return;
+        else if (!setting.zwave.index || !setting.zwave.size) throw new Error(`Missing property in "zwave" at ${driver.id}, ${setting.id}`);
+        else if ( (setting.attr && setting.attr.max) || setting.max ) {
```

Comment:
Same here, if `attr.max` or `setting.max` equals zero, the signed check will not be executed. Again this should never be the case, but might lead to hard to debug issues later on, better to check explicitly if `typeof attr.max === 'number'` in my opinion.

Reply:
Will fix